From 95b012d5388401dd519dbb32a7580c16057764f5 Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Tue, 31 Jan 2017 06:35:52 -0500
Subject: [PATCH 1931/2094] drm/amdgpu: Fix call to amdgpu_vm_bo_update

The 3rd parameter is now a boolean for whether to clear the Page Tables
or not.

Change-Id: I84b2083bd5b2d388181a28b1e819cbb499a966ae
Signed-off-by: Kent Russell <kent.russell@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index 6ac2ba3..9895c8a 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -990,7 +990,7 @@ static int update_gpuvm_pte(struct amdgpu_device *adev,
 	amdgpu_sync_fence(adev, sync, vm->page_directory_fence);
 
 	/* Update the page tables  */
-	ret = amdgpu_vm_bo_update(adev, bo_va, &bo->tbo.mem);
+	ret = amdgpu_vm_bo_update(adev, bo_va, false);
 	if (ret != 0) {
 		pr_err("amdgpu_vm_bo_update failed\n");
 		return ret;
-- 
2.7.4

